### PR TITLE
Include takendown posts on author feed for admins

### PR DESCRIPTION
**Pull request by foysalit**
This allows moderators/admins to review posts from taken down accounts.

### CODE REVIEW

File: packages/bsky/src/api/app/bsky/feed/getAuthorFeed.ts [hunk 1]
```diff
@@ -33,7 +33,11 @@ export default function (server: Server, ctx: AppContext) {
 
       const [result, repoRev] = await Promise.all([
         getAuthorFeed(
-          { ...params, viewer },
+          {
+            ...params,
+            includeSoftDeleted: auth.credentials.type === 'role',
```
Comment (bnewbold) [inline packages/bsky/src/api/app/bsky/feed/getAuthorFeed.ts:38]: just checking, is this supposed to be the string 'role'?

**Comment by bnewbold** [comment]
excited for this! I'm not well-suited to review but strong :heavy_plus_sign: for the functionality, hopefully quickly.

File: packages/bsky/src/api/app/bsky/feed/getAuthorFeed.ts (hunk 1 above)
Comment (foysalit) [reply packages/bsky/src/api/app/bsky/feed/getAuthorFeed.ts:38]: yep. the `credentials.type === 'role'` is equivalent to `credentials.triage === true`

**Review by devinivy — APPROVED** [review]
Great!

**Pull request merged**